sources/HUBComponentImageDataBuilderImplementation.h

```diff
@@ -31,6 +31,9 @@ NS_ASSUME_NONNULL_BEGIN
 /// Concrete implementation of the `HUBComponentImageDataBuilder` API
 @interface HUBComponentImageDataBuilderImplementation : NSObject <HUBComponentImageDataBuilder, NSCopying>
 
+/// Any specific bundle that the builder should use to load local images (defaults to the main bundle)
+@property (nonatomic, weak, nullable) NSBundle *bundle;
```

Comment:
Didn't feel right for the image data builder to retain the bundle, since it shouldn't assume any ownership over it.